testsupport/has/api/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -0,0 +1,123 @@
+// +build !ignore_autogenerated
```

Comment:
Should we include a make target to regenerate CRD-related files for the has api or you think it's not necessary?

Reply:
I don't think that it's needed - this is only as a dummy CRD, we don't need to update it or keep in sync with the real API type / CRD

Reply:
Ok, that's what I figured but just thought I'd ask in case we were planning to update it.